diff --git a/node_modules/animejs/types/index.d.ts b/node_modules/animejs/types/index.d.ts
index edc7ae0..a86951c 100644
--- a/node_modules/animejs/types/index.d.ts
+++ b/node_modules/animejs/types/index.d.ts
@@ -220,7 +220,7 @@ declare function stagger(val: number | string | [
     number | string
 ], params?: StaggerParams): StaggerFunction;
 declare namespace eases {
-    export let linear: (...args?: (string | number)[]) => EasingFunction;
+    export let linear: (...args: (string | number)[]) => EasingFunction;
     export let irregular: (length?: number, randomness?: number) => EasingFunction;
     export let steps: (steps?: number, fromStart?: boolean) => EasingFunction;
     export let cubicBezier: (mX1?: number, mY1?: number, mX2?: number, mY2?: number) => EasingFunction;
