diff --git a/node_modules/vite-plugin-checker/dist/esm/logger.js b/node_modules/vite-plugin-checker/dist/esm/logger.js
index 06bfe1b..6f4a229 100644
--- a/node_modules/vite-plugin-checker/dist/esm/logger.js
+++ b/node_modules/vite-plugin-checker/dist/esm/logger.js
@@ -57,7 +57,7 @@ function diagnosticToTerminalLog(d, name) {
   const position = d.loc ? `${chalk.yellow(d.loc.start.line)}:${chalk.yellow(d.loc.start.column)}` : "";
   return [
     `${levelLabel} ${d.message}`,
-    `${fileLabel + d.id}:${position}${os.EOL}`,
+    `${fileLabel}file://${d.id}:${position}${os.EOL}`,
     d.codeFrame + os.EOL,
     d.conclusion
   ].filter(Boolean).join(os.EOL);
